catalog/schema/enums/workflow_type.yaml

```diff
@@ -0,0 +1,14 @@
+id: https://github.com/galaxyproject/brc-analytics/blob/main/catalog/schema/enums/workflow_type.yaml#
+name: enums_workflow_type
+description: Definition of the workflow type enum.
+
+enums:
+  WorkflowType:
+    description: Set of IDs of workflow categories.
+    permissible_values:
+      VARIANT_CALLING:
+      TRANSCRIPTOMICS:
+      REGULATION:
+      ASSEMBLY:
+      GENOME_COMPARISONS:
+      PROTEIN_FOLDING:
```

Comment:
It might make sense to add a placeholder like `OTHER` so we can generate a valid catalog if a category doesn't match yet ?